python3.hcl

```diff
@@ -1,35 +1,26 @@
 description = "Python is a programming language that lets you work quickly and integrate systems more effectively."
 strip = 1
-binaries = ["install/bin/pip3", "install/bin/python3"]
+binaries = ["install/bin/*"]
 test = "python3 -m pip install flake8"
+
 env = {
-  "PYTHONPYCACHEPREFIX": "${HERMIT_ENV}/.hermit/python/cache",
-  "PIP_CACHE_DIR": "${HERMIT_ENV}/.hermit/python/cache/pip",
-  "PYTHONUSERBASE": "${HERMIT_ENV}/.hermit/python",
-  "PATH": "${HERMIT_ENV}/.hermit/python/bin:${PATH}",
+  PYTHONPYCACHEPREFIX: "${HERMIT_ENV}/.hermit/python/cache",
+  PYTHONUSERBASE: "${HERMIT_ENV}/.hermit/python",
+  PATH: "${HERMIT_ENV}/.hermit/python/bin:${PATH}",
```

Comment:
This has dropped an envar, PIP_CACHE_DIR

Reply:
Ended up reverting the formatting